home/forms/auth_forms.py

```diff
@@ -42,12 +42,12 @@ def __init__(self, *args, **kwargs):
     def generate_layout(self):
         return Layout(
             Div(
-                Field('username', placeholder="Username", wrapper_class='mb-0'),
+                Field('username', placeholder="Username", wrapper_class='mb-0', id="id_login_username"),
```

Comment:
prefixing with `id` is redundant and bad style imo, would prefer just `login_username`. Similarly elsewhere.

Just thinking about it in terms of a uniqueness perspective, if you prefix every id with `id_`, then any ids that clashed before would still clash, so you've gained nothing.

Reply:
the ids now contain the name of the form they belong to so they're all unique. If you look at the username field for the register form, you'll see that the id is "id_register_username". 
I'll remove the `id_` prefix

Reply:
I realize the ids are all unique in this pr, I was just explaining why I don't think prefixing ids with `id_` is a fruitful exercise (because doing so everywhere doesn't "add" any uniqueness).